controller/modules/equipment/controller.go

```diff
@@ -24,6 +25,16 @@ func New(c controller.Controller) *Controller {
 }
 
 func (c *Controller) Setup() error {
+	eqs, err := c.List()
+	if err != nil {
+		log.Println("ERROR: equipment subsystem: failed to list equipment outlets during Setup. Error:", err)
+	}
+	for _, eq := range eqs {
+		if eq.StayOffOnBoot && eq.On {
```

Comment:
This check  should be moved in start method , setup is used in the very begining, List can fail if the storage bucket is not setup.
```go
for _, eq := range eqs {
if eq.StayOffOnBoot { 
eq.On = false
}
c.Update(eq.ID, eq)
}

```

Reply:
Moved to Start method and simplified StayOffOnBoot logic as requested by R. Dey.